README.md

```diff
@@ -1,2 +1,18 @@
 # Hello-Microverse
-Microverse project 
+Microverse project
+
+### Built With
+Html<br />
+CSS
+
+### Author
+<a href="https://github.com/kusiLaw">Github</a> \
+<a href="https://twitter.com/kusilaw">Twitter</a> \
+<a href="https://www.linkedin.com/in/lawrence-kusi-55a662104">Linkedin</a>
+
+### Acknowledgments
+Thanks to Microverse
+
+### License
+This Project is MIT licensed
```

Comment:
- kindly update the MIT license link. You can use the MIT from the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and add it to the root directory of your project to make your readme appear more professional.